pinot-core/src/main/java/org/apache/pinot/core/segment/index/ColumnMetadata.java

```diff
@@ -39,7 +39,27 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import static org.apache.pinot.core.segment.creator.impl.V1Constants.MetadataKeys.Column.*;
+import static org.apache.pinot.core.segment.creator.impl.V1Constants.MetadataKeys.Column.BITS_PER_ELEMENT;
```

Comment:
Are these new imports needed?

Reply:
Yes they were all used before. Formatting removed the wildcard import